src/get-relayer-address.ts

```diff
@@ -0,0 +1,26 @@
+import { Account } from "viem";
+
+import { config } from "./config";
+import { deriveRelayerAccount } from "./utils";
+import getSecret from "./get-secret";
+
+async function main() {
```

Comment:
naming-wise i'd suggest to change this to sth like `get-relay-trigger-wallet.ts` or `get-relay-trigger-wallet-address.ts`

because we use the term **relayer address** everywhere else to mean the address of a relayer contract

Reply:
changed to relayer-signer